lib/play_game.py

```diff
@@ -3,27 +3,38 @@
 
 
 class PlayGame:
+
+    current_player = "X"
+    next_player = "O"
+
     def __init__(self, board, ui):
         self.board = board
         self.ui = ui
 
     def start(self):
         self.ui.display("Welcome to TIC TAC TOE!")
         self.board.display_board()
+        self.ui.display(self.current_player + " goes first")
         while not self.board.board_full(self.board.board):
-            self.turn()
+            self.turn(self.current_player)
+            self.switch_players()
 
-    def turn(self):
+    def turn(self, mark):
         self.ui.display("Choose a number between 1 and 9:")
         input_number = input()
```

Comment:
This is another place where you should lean on the UI class, collecting user input. That UI class should be a wrapper for the print() and input() methods, which will help you in testing for mocking out those functions.